### DISCUSSION TITLE
Starting FrankenPHP as a service without docker

### DISCUSSION THREAD

**Discussion by luisalvarado** [php/frankenphp#943] [General]
I am having a bit of a confusion on how to start FrankenPHP as a service on Ubuntu (without using docker or any container manager). For example, similar to caddy that has a service option once installed from the PPA that basically makes it into a service, saving me the guessing work for it.

Is this something that can be accomplished with FrankenPHP and if so, how since I can not find like a simple step by step tutorial for this idea. Thank you.

**Comment by dunglas** [comment]
We currently don't provide a PPA (but contributions are welcome!). However, if you compile the project from sources or install the static binary, it's totally possible to make a standard Ubuntu service (you can reuse [the Caddy systemd templates](https://caddyserver.com/docs/running#unit-files)).

**Comment by luisalvarado** [comment]
To avoid creating too many tickets for questions, I was able to use the installer and that works great. I also had to configure the systemd part to adapt to what I needed, but the last thing I need is, without using docker, does frankenphp support connecting to mysql via its internal php (To mariadb server)? I am asking because in the Caddyfile I have to have a fallback like this so that fpm connects to mariadb, but if I use the internal php of frankenphp it does not connect

handle {
    php_fastcgi unix//run/php/php-fpm.sock
    file_server
}

Just trying to understand if I can skip php fpm and only use frankenphp. Thank you very much.
